### DISCUSSION TITLE
How to attach simple screenshot to the Allure report ?

### DISCUSSION THREAD

**Discussion by NebelskyiDmytro** [allure-framework/allure2#2788] [Questions & Support] [answered]
Hello community,

Can someone please help me to attach my screenshot to the Allure report ? Where can I get latest documentation for it ?

I am using JavaScript + Playwright. I have tried almost everything, nothing works, everytime I am getting "allure.attachment is not a function" and same errors like this, I have tried a lot of functions.

**Comment by todti** [comment, marked as answer]
@NebelskyiDmytro Hi! Could you try doing [this](https://allurereport.org/docs/playwright/#attach-screenshots-and-other-files) ?

**Reply by NebelskyiDmytro** [reply]
Thank you very much ! Sorry for such stupid question.

**Comment by todti** [comment]
If your file needs to be uploaded from a specific directory, you should use `attachmentPath` instead of `attachment`. If you still encounter any issues, please send an example. 😊
